cli/tests/test_net.rs

```diff
@@ -167,7 +167,7 @@ fn test_port_config() {
             autoneg: false,
             fec: LinkFec::None,
             link_name: String::from("phy0"),
-            lldp_service_config_id: Uuid::new_v4(),
+            lldp_link_config_id: None,
```

Comment:
@Nieuwejaar this seems to differ from what you have in #808--it looks like you had a required field

Reply:
You're right.  This is fine.  Even if the LLDP stuff is configured, the `system networking switch-port-settings show` doesn't include it in the output, so it doesn't matter whether this data structure is set up or not.